pkg/httputils/http_utils_test.go

```diff
@@ -2,24 +2,52 @@ package httputils
 
 import (
 	"net/http"
+	"strings"
 	"testing"
 )
 
 func TestBoolValue(t *testing.T) {
-	type args struct {
-		r *http.Request
-		k string
-	}
 	tests := []struct {
-		name string
-		args args
-		want bool
+		name    string
+		testStr string
+		want    bool
 	}{
-		// TODO: Add test cases.
+		{"test normal string", "normal string", true},
+		{"test ''", "", false},
+		{"test 0", "0", false},
+		{"test no", "no", false},
+		{"test No", "No", false},
+		{"test No", "No", false},
+		{"test NO", "NO", false},
+		{"test false", "false", false},
+		{"test False", "False", false},
+		{"test FALSE", "FALSE", false},
+		{"test none", "none", false},
+		{"test None", "None", false},
+		{"test NONE", "NONE", false},
+		// test trim
+		{"test '  '", "  ", false},
+		{"test trim 0", "  0  ", false},
+		{"test trim no", "  no  ", false},
+		{"test trim No", "  No ", false},
+		{"test trim NO", "  NO ", false},
+		{"test trim false", " false ", false},
+		{"test trim False", " False  ", false},
+		{"test trim FALSE", "  FALSE ", false},
+		{"test trim none", "  none ", false},
+		{"test trim None", "  None  ", false},
+		{"test trim NONE", "  NONE  ", false},
 	}
 	for _, tt := range tests {
+		request, _ := http.NewRequest(
+			"POST",
+			"http://test",
+			strings.NewReader("name="+tt.testStr))
```

Comment:
I think define a var, boolName = "name" is better, because you also use it in the above code, just in case they're divergent.